timon/plugins/http_server/views.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+
+# #############################################################################
+# Copyright : (C) 2023 by MHComm. All rights reserved
+#
+# __author__ = "Quentin Laymajoux"
+# __email__ = "info@mhcomm.fr"
+#
+# Name       : timon.plugins.http_server.views
+"""
+Summary: The http server plugin views and routes
+"""
+# #############################################################################
+import logging
+
+from quart_trio import QuartTrio
+
+from timon.probes.probe_if import mk_probe
+
+logger = logging.getLogger(__name__)
+
+app = QuartTrio(__name__)
+
+KNOWN_ROUTES = {
+    "/resources/": "returns a list of all resources and their availability",
+    "/heap/": "returns the lenght of the heap",
+    "/probes/<?probename>/run/": ("force run the probename and returns "
```

Comment:
forces to run the probe with name probe_name and returns its result.

Not sure this is a good idea. I try to explain.

Imagine you want to force running 100 probes (via a UI)
then you might potentially not get all results within 2 minutes and your browser would time out.

I'd suggest:

"schedules to run the probe as soon as possible"

We should get results via websockets or just by polling.

Or at least have a blocking and a non blocking version of this url

Reply:
Use the semaphore like a normal run + Added a reschedule url